demo/python/basic/secure-xgboost-demo.py

```diff
@@ -51,6 +51,11 @@
 # Save model to a file
 booster.save_model(HOME_DIR + "demo/python/basic/modelfile.model")
 
+print("Loading model")
+# Load model from file
```

Comment:
Do we support load model now? If I remember correctly, adding an enclave symmetric key should enable this?

Reply:
Exactly.

Reply:
Cool! Does the load model code still need some work, or can it be uncommented?

Reply:
It works, uncommenting it.